README.md

```diff
@@ -17,82 +17,125 @@
 AMQ producer example
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://router.router.svc.cluster.local"
-            - "--api-port=8089"
+            - "--transport-host=amqp://amq-router.amq-router.svc.cluster.local"
+            - "--api-port=9085"
 ```
 
 AMQ consumer example              
-```json
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://router.router.svc.cluster.local"
-            - "--api-port=8089"
+            - "--transport-host=amqp://amq-router.amq-router.svc.cluster.local"
+            - "--api-port=9085"
 ```
-#### HTTP Protocol
-##### Producer
-CloudEvents HTTP Protocol will be enabled based on  url in `transport-host`.
+### HTTP Protocol
+#### Producer
+CloudEvents HTTP Protocol will be enabled based on url in `transport-host`.
 If HTTP is identified then the publisher will start a publisher rest service, which is accessible outside the container via k8s service name.
-The Publisher service will have the ability to register  consumer endpoints to publish events.
+The Publisher service will have the ability to register consumer endpoints to publish events.
 
-HTTP producer example  
-```json
+The transport URL is defined in the format of 
+```yaml
+- "--transport-host=$(TRANSPORT_PROTOCAL)://$(TRANSPORT_SERVICE).$(TRANSPORT_NAMESPACE).svc.cluster.local:$(TRANSPORT_PORT)"
+```
+
+HTTP producer example
+```yaml
  - name: cloud-event-sidecar
           image: quay.io/redhat-cne/cloud-event-proxy
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
-            - "--api-port=8089"
+            - "--api-port=9085"
 
+```
+The event producer uses a `pubsubstore` to store Subscriber information, including clientID, consumer service endpoint URI, resource ID etc. These are stored as one json file per registered subscriber. The `pubsubstore` needs to be mounted to a persistent volume in oder to survive a pod reboot.
+
+Example for configuring persistent storage
+
+```yaml
+     spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      serviceAccountName: hw-event-proxy-sa
+      containers:
+        - name: cloud-event-sidecar
+          volumeMounts:
+            - name: pubsubstore
+              mountPath: /store
+      volumes:
+        - name: pubsubstore
+          emptyDir: {}
+          persistentVolumeClaim:
+            claimName: cloud-event-proxy-store
+```
 
+Example PersistentVolumeClaim
```

Comment:
Don't you want PersistentVolume definition

Reply:
Customer is free to choose how the PersistentVolume is created. For example when the Local Storage operator is installed, PV can be created automatically by the Local Storage operator.